### PR TITLE
Fix supplier invoice problem

### PR DESCRIPTION
**Pull request by SylvainLegrand**
When you modify a supplier invoice line value the description of the modified line is erased.

### CODE REVIEW

File: script/interface.php [hunk 1]
```diff
@@ -185,7 +185,7 @@ function _updateObjectLine($objectid, $objectelement,$lineid,$column, $value) {
         }
 		elseif ($objectelement == "FactureFournisseur")
         {
-            $res = $o->updateline($lineid, $line->desc, $price, $line->tva_tx, $line->localtax1_tx, $line->localtax2_tx, $qty, $line->fk_product, 'HT', $line->info_bits, $line->product_type, $remise_percent, false, $line->date_start, $line->date_end, $line->array_options, $line->fk_unit, $line->multicurrency_subprice, $line->ref_supplier);
+            $res = $o->updateline($lineid, $line->description, $price, $line->tva_tx, $line->localtax1_tx, $line->localtax2_tx, $qty, $line->fk_product, 'HT', $line->info_bits, $line->product_type, $remise_percent, false, $line->date_start, $line->date_end, $line->array_options, $line->fk_unit, $line->multicurrency_subprice, $line->ref_supplier);
```
Comment (atm-john) [inline script/interface.php:188]: do you know version of Dolibarr who need this change ? is for retro-compatibility.

Comment (atm-john) [reply script/interface.php:188]: same as #39

**Comment by SylvainLegrand** [comment]
The problem was seen on version 14

File: script/interface.php (hunk 1 above)
Comment (atm-john) [inline script/interface.php:188]:
```suggestion
            $lineDesc = !empty($line->description)?$line->description:$line->desc;// for compatibility Dolibarr V14 and above
            $res = $o->updateline($lineid, lineDesc, $price, $line->tva_tx, $line->localtax1_tx, $line->localtax2_tx, $qty, $line->fk_product, 'HT', $line->info_bits, $line->product_type, $remise_percent, false, $line->date_start, $line->date_end, $line->array_options, $line->fk_unit, $line->multicurrency_subprice, $line->ref_supplier);
```

Comment (SylvainLegrand) [reply script/interface.php:188]: Nice

Comment (SylvainLegrand) [reply script/interface.php:188]: Please don't forget the $ before lineDesc variable ! ;)

**Pull request merged**